contracts/cw20-stake-external-rewards/src/contract.rs

```diff
@@ -164,6 +196,51 @@ pub fn execute_fund_native(
     }
 }
 
+pub fn execute_withdraw(
+    deps: DepsMut,
+    env: Env,
+    info: MessageInfo,
+) -> Result<Response<Empty>, ContractError> {
+    // only the owner can withdraw
+    let config = CONFIG.load(deps.storage)?;
```

Comment:
should the manager be able to withdraw as well?

Reply:
would be interested in @ben2x4's thoughts here. i'd imagine we want to only allow the owner to do this.

Reply:
Agree with @ezekiiel here.